bin/plotting/pycbc_page_foundmissed

```diff
@@ -9,16 +9,30 @@ import pycbc.pnutils
 from pycbc import init_logging
 from pycbc.detector import Detector
 
+labels={'mchirp': 'Chirp Mass',
+        'mtotal': 'Total Mass',
+        'mass_ratio': 'Mass Ratio',
+        'decisive_distance': 'Decisive Distance (Mpc)',
+        'dec_chirp_distance': 'Decisive Chirp Distance (Mpc)',
+        'min_eff_distance': 'Minimum Effective Distance (Mpc)',
+        'min_eff_chirp_distance': 'Minimum Effective Chirp Distance (Mpc)',
+        'chirp_distance': 'Injected Chirp Distance (Mpc)',
```

Comment:
I would still remove the 'Injected' here as the other labels seem to do OK without it

Reply:
Fixed now